kits/subscription-audit/README.md

```
@@ -0,0 +1,89 @@
+
+### 2. Install & Run
+
+```bash
+npm install
+npm run dev
+# Open http://localhost:3000
+```
+
+---
+
+## 📂 Repo Structure
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)